core/arch/arm/include/mm/tee_pager.h

```diff
@@ -94,4 +94,16 @@ void tee_pager_release_zi(vaddr_t vaddr, size_t size);
  */
 void *tee_pager_request_zi(size_t size);
 
+/*
+ * Statistics on the pager
+ */
+struct tee_pager_stats {
+	size_t hidden_hits;
+	size_t ro_hits;
+	size_t rw_hits;
+	size_t zi_released;
+	size_t npages;		/* number of load pages */
+	size_t npages_all;	/* number of pages */
```

Comment:
- Is `npages` really needed? It seems it is kept in sync with `tee_pager_npages`. Why would we want to maintain the same value in two places? And, I would say that `tee_pager_npages` misses a qualifier: what kind of page is that exactly? `s/tee_pager_npages/npages_used/` maybe?
- (Nit-picking) Then `npages_all` could as well be called `npages` especially if `tee_pager_npages` is renamed.